### ISSUE TITLE
Feature request: Pagination

### ISSUE THREAD

**Issue by adam-devapp** [eileenmcnaughton/nz.co.fuzion.accountsync#57]
A few clients have requested adding pagination to the contact sync page. Current the page only shows the first 10 contacts. 
![image](https://user-images.githubusercontent.com/13082954/113794842-9968df00-978a-11eb-8b0a-01068e9acfb2.png)

**Comment by eileenmcnaughton** [comment]
Yeah there is a way to add pagination to an angular page (I use it in the deduper) - it takes a bit of work though. Resolving any of the first 10 obviously takes it off the page bringing the next one on so if they do take action to exclude etc them they will get to see them all

**Comment by mattwire** [comment]
@devappsoftware See https://github.com/eileenmcnaughton/nz.co.fuzion.accountsync/pull/61
